lineapy/plugins/jinja_templates/kubeflow_dockerfile.jinja

```diff
@@ -0,0 +1,28 @@
+# Be sure to build this docker file with the following command
+# docker build -t data_housing_pipeline:lineapy -f data_housing_pipeline_Dockerfile .
+
+FROM python:{{ python_version }}
+
+RUN mkdir /tmp/installers
+WORKDIR /tmp/installers
+
+# Copy all the requirements to run current DAG
+COPY ./{{ pipeline_name }}_requirements.txt ./
+
+# Install kubeflow python sdk
+RUN apt update
+RUN pip install kfp
+
+# Install required libs
+RUN pip install -r ./{{ pipeline_name }}_requirements.txt
+
+WORKDIR /home
+COPY . .
```

Comment:
if we know that we'll only need {{pipeline_name}}_module.py and _dag.py maybe we can restrict copy to just those two so that we dont copy all the other files and airflow junk that we have in the same folder. not required, ok to skip for this PR.

Reply:
Changed